saws-s3/src/main/scala/com/ambiata/saws/s3/S3.scala

```diff
@@ -13,7 +13,7 @@ object S3 {
     S3Action(_.listBuckets.toList).onResult(_.prependErrorMessage(s"Could access the buckets list"))
 
   def isS3Accessible: S3Action[Unit] =
-    listBuckets.map(_ => ()).orElse(S3Action.fail("S3 is not accessible"))
+    listBuckets.void.orElse(S3Action.fail("S3 is not accessible").void)
```

Comment:
Doesn't matter that much, but you can probably just void this once on the outside.